src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Compilation.targets

```diff
@@ -139,10 +154,12 @@ Copyright (c) .NET Foundation. All rights reserved.
          NoLogo="$(NoLogo)"
          NoStandardLib="$(NoCompilerStandardLib)"
          NoWin32Manifest="$(NoWin32Manifest)"
+         NullableContextOptions="$(NullableContextOptions)"
          Optimize="$(Optimize)"
          Deterministic="$(Deterministic)"
          PublicSign="$(PublicSign)"
          OutputAssembly="@(RazorIntermediateAssembly)"
+         PdbFile="$(PdbFile)"
```

Comment:
We'd specifically remove this - https://github.com/aspnet/AspNetCore-Tooling/blob/519d67bfb131caf64903ae35c95f5e6f38e1b6bf/src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Compilation.targets#L44